types/katex/index.d.ts

```diff
@@ -140,24 +140,26 @@ export class ParseError implements Error {
     position: number;
 }
 
-export default class katex {
+declare const _default: {
     /**
      * Renders a TeX expression into the specified DOM element
      * @param tex A TeX expression
      * @param element The DOM element to render into
      * @param options KaTeX options
      */
-    static render(tex: string, element: HTMLElement, options?: KatexOptions): void;
+    render: (tex: string, element: HTMLElement, options?: KatexOptions) => void,
 
     /**
      * Renders a TeX expression into an HTML string
      * @param tex A TeX expression
      * @param options KaTeX options
      */
-    static renderToString(tex: string, options?: KatexOptions): string;
+    renderToString: (tex: string, options?: KatexOptions) => string,
 
     /**
      * KaTeX error, usually during parsing.
      */
-    static ParseError: typeof ParseError;
+    ParseError: typeof ParseError,
 }
+
+export default _default;
```

Comment:
The rename from `katex` to `_default` will affect auto-imports in TS Server. Conventional usage seems to be importing as `katex`, so the local name of the symbol should remain that way.